src/pages/index.tsx

```diff
@@ -92,17 +93,34 @@ export default function Home() {
         </TitleAndSwiperContainer>
         <BoxBadgeTypes>
           <SectionTitle>Types of badges</SectionTitle>
-
           <BadgeTypeDescription
             tittle={'Off-chain Badges'}
-            body={`These badges will be an on-chain representation of off-chain information. There are going to be strategies developed ad-hoc for each type of badge. In order to mint these badges evidence has to be presented and pass throug a curation process in Kleros.`}
+            body={
+              <div>
+                Are badges generated and backed by{' '}
+                <CustomLink
+                  href={'https://kleros.gitbook.io/docs/products/curate'}
+                  target={'_blank'}
+                >
+                  Kleros
+                </CustomLink>
+                . In order to mint these badges evidence has to be presented and
+                pass through a curation process.
+              </div>
+            }
             image={svg2}
             adornment={adornmentKleros}
           />
 
           <BadgeTypeDescription
             tittle={'Third-party Badges'}
-            body={`Are badges generated by a public or private entity. Entities that are willing to generate private badges will need to apply and be allowed before they can start emitting badges.`}
+            body={
+              <div>
+                Are badges generated and backed by a public or private entity.
+                Entities that are willing to generate these badges will need to
+                apply and be allowed before they can start emitting them.
```

Comment:
The last part is not right, we don't want to avoid entities emitting their own badges. The only restriction is that if they are not validated, they won't be able to mint badges to addresses, they will only be able to whitelist them